test_scripts/AppServices/UnpublishAppService/002_HMI_success_flow.lua

```diff
@@ -0,0 +1,64 @@
+---------------------------------------------------------------------------------------------------
```

Comment:
Missing Mobile success flow. Both tests should be added to the test set.

Reply:
Could also use a test case for if an app tries to unpublish a service that it didn't publish.